src/middlewared/middlewared/plugins/truecommand/wireguard.py

```diff
@@ -127,7 +127,12 @@ async def start_truecommand_service(self):
             if Status(config['api_key_state']) == Status.CONNECTED and all(
                 config[k] for k in ('wg_private_key', 'remote_address', 'endpoint', 'tc_public_key', 'wg_address')
             ):
-                await self.middleware.call('service.start', 'truecommand')
+                if not await self.middleware.call('service.started', 'truecommand'):
```

Comment:
Is this required ? Just want to account for random race conditions

Reply:
I'm not sure. Testing without it.

Reply:
Working fine now, but needs a one line patch in the port. I'll link that to this issue.